### DISCUSSION TITLE
Load Cell Temp

### DISCUSSION THREAD

**Discussion by MoonDoggie97** [xblax/flashforge_ad5m_klipper_mod#123] [Q&A]
When the printer is in standby doing nothing the load cell temperature is going up to 600+ Celsius is this normal? Will this kill the device quicker

**Comment by consp** [comment]
> When the printer is in standby doing nothing the load cell temperature is going up to 600+ Celsius is this normal? Will this kill the device quicker

"Temperature" is just the value of the strain gauges. It's not an actual temperature. 

When it goes idle the bed contracts (due to reducing in temperature). The strain gauge thus change shape and send a signal. This is completely normal and it's zeroed every time you auto-bed-level it (and then it should be heated) so idle will be non-zero when idle and cold.

**Comment by GosuKitten** [comment]
I came in here about to make the same post and im glad it's been answered. that's curious, and good to know that the cell value isn't actual celsius. I was wondering isn't that going to kill the component too heh. Thank you for answering this for us @consp

**Reply by consp** [reply]
No problem, also moved into Q&A since it's not an issue but might need to show up for others.

**Comment by consp** [comment]
To add if someone asks about the TVOC sensor value on the Pro: Not a temperature either. It's a "value" indication in the range 0-500 where 0 is "none" and 500 is "a lot" (that's as good as it gets)

The strain gauges are 0-2000 if I'm not mistaken.
